cpp/Platform.Collections/Stacks/CStack.h

```diff
@@ -0,0 +1,24 @@
+#pragma once
+
+namespace Platform::Collections::Stacks
+{
+    template<typename TSelf,typename TElement>
+    concept CStack =
+        requires(TSelf& self,TElement item) 
+        {
+            { self.empty() } -> std::same_as<bool>;
```

Comment:
Does `std::stack` has a non const `empty` method?

Reply:
Remove it